kaznet/apps/main/serializers/submissions.py

```diff
@@ -13,15 +14,34 @@
                                           LABEL_LOCATION_ID, LABEL_TASK_ID)
 from kaznet.apps.main.models import Submission
 from kaznet.apps.main.serializers.base import GenericForeignKeySerializer
-from kaznet.apps.main.serializers.bounty import SerializableAmountField
 
 
+class SerializableDateField(serializers.Field):
+    """
+    Custom Field submission_time
+    """
+
+    def to_representation(self, value):
+        """
+        Custom to submission_date field
+        """
+        return value.astimezone(pytz.timezone('Africa/Nairobi')).replace(
+            microsecond=0).isoformat()
+
+    def to_internal_value(self, data):
```

Comment:
Why do we need to include this method?

Reply:
The method to_interval_value is abstract in serializers.Field. so it must be included